security_mvp/src/emitters/rabbitmq_emitter.py

```
@@ -0,0 +1,41 @@
+
+class RabbitMQEmitter(QueueEmitter):
+    def __init__(self, url: str, exchange: str) -> None:
+        self.url = url
+        self.exchange = exchange
+        self.logger = logging.getLogger(__name__)
+        self._connection: Optional["pika.BlockingConnection"] = None
+        self._channel: Optional["pika.channel.Channel"] = None
+
+    def _connect(self) -> None:
+        if pika is None:
+            raise RuntimeError("pika is required for RabbitMQEmitter")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)